generator/generator.go

```diff
@@ -20,13 +20,38 @@ type featureHelpers struct {
 	feature int
 }
 
-type ObjectSet map[protogen.GoIdent]bool
+type ObjectSet struct {
+	wildcards map[string]struct{}
+	mp        map[protogen.GoIdent]bool
+}
+
+func NewObjectSet() ObjectSet {
+	return ObjectSet{
+		mp:        map[protogen.GoIdent]bool{},
+		wildcards: map[string]struct{}{},
+	}
+}
 
 func (o ObjectSet) String() string {
 	return fmt.Sprintf("%#v", o)
 }
 
+func (o ObjectSet) Contains(g protogen.GoIdent) bool {
+	for wildcard := range o.wildcards {
+		if Match(wildcard, fmt.Sprintf("%s.%s", string(g.GoImportPath), g.GoName)) {
+			return true
+		}
+	}
+
+	return o.mp[g]
+}
+
 func (o ObjectSet) Set(s string) error {
+	if strings.Contains(s, "*") {
```

Comment:
This is not great, and won't work once we use `path.Match` because those wildcards also support `?`. I think we should assume that all patterns _are_ wildcards and just attempt matching accordingly.